fe1-web/src/features/rollCall/objects/RollCall.ts

```diff
@@ -58,7 +58,7 @@ export class RollCall {
   }
 
   public get end() {
-    return this.closedAt || this.proposedEnd;
```

Comment:
are you sure this won't have side effects? 🤔

Reply:
No, I am also a little scared what could break but so far it seems to work pretty well. There was only a few places where I saw this getter being used and there it should be fine..